README.md

```diff
@@ -12,6 +12,8 @@ npm i orbit-db-access-controllers
 
 ## Usage
 
+The default access control is set so that the initial user is the only one who has access.
```

Comment:
Hmm... maybe

> By default, access is granted only to the user who created  the AccessController.

Reply:
How does the following sound: By default, if no write-array is specified in options, the access control is set so that the initial user is the only one who has access (specified by the `identity` property of the `orbitdb` instance given in the argument). For the Ethereum-based contract example, the account which deploys the contract is initially given access.

Maybe wording could use another iteration :sweat_smile:

Reply:
That works for me! I think that's fine. It is a bit complicated by nature.

Reply:
Thanks Richard! Only comment is that the example "below" is not an Ethereum example, the ethereum example is in src/contract-access-controller.js. The one below `Usage` is just an example of adding a custom access-controller

Reply:
Removed. Good call.